deployment/ccip/changeset/aptos/cs_add_token_pool.go

```diff
@@ -214,6 +218,20 @@ func (cs AddTokenPool) Apply(env cldf.Environment, cfg config.AddTokenPoolConfig
 	}, nil
 }
 
+func isTokenOwnedByMCMS(deps operation.AptosDeps, cfgTokenAddress aptos.AccountAddress) (bool, error) {
+	if cfgTokenAddress == (aptos.AccountAddress{}) {
+		// Token cfg not provided, so token is newly deployed and owned by MCMS
+		return true, nil
+	}
+	mcmsAddress := deps.CCIPOnChainState.AptosChains[deps.AptosChain.Selector].MCMSAddress
+	mcmsContract := mcmsbind.Bind(mcmsAddress, deps.AptosChain.Client)
+	isOwned, err := mcmsContract.MCMSRegistry().IsOwnedCodeObject(nil, cfgTokenAddress)
+	if err != nil {
```

Comment:
One question: Should I consider some of those errors `E_ADDRESS_NOT_REGISTERED` or `E_INVALID_CODE_OBJECT` as just not owned by MCMS or should the Token be always registered?

https://github.com/smartcontractkit/chainlink-aptos/blob/8ef0e7547ab26cc0dec9b579f1b0876a22a3a82c/contracts/mcms/mcms/sources/mcms_registry.move#L179-L202